probes/requiresCodeOwnersReview/impl_test.go

```diff
@@ -0,0 +1,315 @@
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &falseVal,
+								},
+							},
+						},
+					},
+					CodeownersFiles: []string{"file"},
+				},
+			},
+			outcomes: []finding.Outcome{
+				finding.OutcomePositive, finding.OutcomeNegative,
+			},
+		},
+		{
+			name: "Requires code owner reviews on 1/2 branches - 1 - without files",
+			raw: &checker.RawResults{
+				BranchProtectionResults: checker.BranchProtectionsData{
+					Branches: []clients.BranchRef{
+						{
+							Name: &branchVal1,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &trueVal,
+								},
+							},
+						},
+						{
+							Name: &branchVal2,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &falseVal,
+								},
+							},
+						},
+					},
+					CodeownersFiles: []string{},
+				},
+			},
+			outcomes: []finding.Outcome{
+				finding.OutcomeNegative, finding.OutcomeNegative,
+			},
+		},
+		{
+			name: "Requires code owner reviews on 1/2 branches - 2 - with files",
+			raw: &checker.RawResults{
+				BranchProtectionResults: checker.BranchProtectionsData{
+					Branches: []clients.BranchRef{
+						{
+							Name: &branchVal1,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &falseVal,
+								},
+							},
+						},
+						{
+							Name: &branchVal2,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &trueVal,
+								},
+							},
+						},
+					},
+					CodeownersFiles: []string{"file"},
+				},
+			},
+			outcomes: []finding.Outcome{
+				finding.OutcomeNegative, finding.OutcomePositive,
+			},
+		},
+		{
+			name: "Requires code owner reviews on 1/2 branches - 2 - without files",
+			raw: &checker.RawResults{
+				BranchProtectionResults: checker.BranchProtectionsData{
+					Branches: []clients.BranchRef{
+						{
+							Name: &branchVal1,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &falseVal,
+								},
+							},
+						},
+						{
+							Name: &branchVal2,
+							BranchProtectionRule: clients.BranchProtectionRule{
+								RequiredPullRequestReviews: clients.PullRequestReviewRule{
+									RequireCodeOwnerReviews: &trueVal,
+								},
+							},
+						},
+					},
+					CodeownersFiles: []string{},
+				},
+			},
+			outcomes: []finding.Outcome{
+				finding.OutcomeNegative, finding.OutcomeNegative,
+			},
+		},
+		{
+			name: "Requires code owner reviews on 0/2 branches - with files",
```

Comment:
not avail

Reply:
https://github.com/ossf/scorecard/pull/3691/commits/d7c16773ae85b5c49cffe37a49f8330e60bdd631